.github/workflows/stale.yml

```diff
@@ -12,5 +12,5 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'This issue is stale, because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days'
```

Comment:
```suggestion
        stale-issue-message: 'This issue is stale, because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 5 days'
```